dashboard/app/api.go

```diff
@@ -855,6 +855,9 @@ func (crash *Crash) UpdateReportingPriority(c context.Context, build *Build, bug
 	if crash.Title == bug.Title {
 		prio += 1e8 // prefer reporting crash that matches bug title
 	}
+	if _, mgrConfig := activeManager(crash.Manager, bug.Namespace); mgrConfig != nil {
+		prio += int64(mgrConfig.Priority * 1e4)
```

Comment:
I would shift it to into positive range.
Currently all signals take 1 decimal digit, so it makes it easy to read these numbers and see what decimal digits are already used.
Negative values make both things harder.
With negative value we will borrow from the next set signal, or if no larger signals set, make the whole value negative, right?
I would use 1 decimal digit 0..9. It makes it impossible to use -5..5 b/c that's 11 values. That's why I proposed -3..3.

Reply:
> With negative value we will borrow from the next set signal, or if no larger signals set, make the whole value negative, right?

Good point!
I've updated the PR to use -3..3. And added an admin method to force refresh priorities for crashes mentioned in bug origin testing (for other crashes we don't really care that they will temporarily move 1-2 positions in the list).